milatools/utils/disk_quota.py

```diff
@@ -92,8 +92,8 @@ def _parse_lfs_quota_output(
         _grace_files,
     ) = values_line_parts
 
-    used_gb = int(used_kbytes.strip()) / (1024**2)
-    max_gb = int(limit_kbytes.strip()) / (1024**2)
+    used_gb = int(used_kbytes.rstrip("*Gk")) / (1024**2)
+    max_gb = int(limit_kbytes.rstrip("*Gk")) / (1024**2)
```

Comment:
Spaces at the end of the string need to be removed before you can call rstrip with those characters.
For example `"123456* ".rstrip("*Gk")` gives `"123456* "`, but `"123456* ".strip().rstrip("*Gk")` gives the answer we want: `"123456"`

```suggestion
    used_gb = int(used_kbytes.strip().rstrip("*Gk")) / (1024**2)
    max_gb = int(limit_kbytes.strip().rstrip("*Gk")) / (1024**2)
```